app/jobs/purge_job.rb

```diff
@@ -25,13 +24,14 @@ def perform(*_args)
                      laboratory_personnel was_in_health_care_facility_with_known_cases
                      healthcare_personnel crew_on_passenger_or_cargo_flight white
                      black_or_african_american american_indian_or_alaska_native asian
-                     native_hawaiian_or_other_pacific_islander ethnicity]
+                     native_hawaiian_or_other_pacific_islander ethnicity purged
+                     continuous_exposure]
       attributes -= whitelist
```

Comment:
Another small thing - can we change this variable name and update the comment above where it is declared as part of this PR? "Whitelist" is not a great term and one that should be steered away from in common vernacular anyway. How about `allowed_list`?

Reply:
sounds good